library/std/src/env.rs

```diff
@@ -1045,6 +1045,45 @@ pub mod consts {
     #[stable(feature = "env", since = "1.0.0")]
     pub const ARCH: &str = env!("STD_ENV_ARCH");
 
+    /// An identifier used to select the platform for which the standard library was built
```

Comment:
This constant isn't used to select anything, right? I think the text here should resemble the docs for `pub const ARCH` above, so something like:

> A string describing ...

Reply:
I've meant it in the sense of "std has been compiled with --target=this-string"

Reply:
Right, then I think this is clearer

```suggestion
    /// A string describing the platform for which the standard library was built
```

Reply:
Perhaps "`--target` argument passed to rustc for this compilation"? That seems like it ought to match 1:1 to what we're returning.

I'm not sure I'm convinced that this should have HOST in its name -- we don't have that for any of the other constants in this file, and they're all derived from the same source. For a rustc `--print` argument it makes more sense to differentiate, but here I'm not convinced.

Reply:
`std::env::consts::TUPLE` doesn't feel easily confusable with the tuple types --  and any such confusion should clear up quickly when actually seeing it's a const with a str value.

I'd personally argue "TARGET" or "HOST" is also not unreasonable naming, given the rest of the constants in that module.